portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -36,16 +36,16 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-    ArrayList<String> strings = new ArrayList<>();
+   private int numOfComments = 1;
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
 
     Query query = new Query("Comments").addSort("comment", SortDirection.DESCENDING);
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
-
     ArrayList<String> toSendOver = new ArrayList<>();
+    toSendOver.add(String. valueOf(numOfComments));
```

Comment:
I dont think you need to send the length, just send the array list

Reply:
This is the old code. It's fix now.

Reply:
sorry, I was using github reviews incorrectly. My analogies to internal code tooling were wrong. I talked to a colleague and I understand better